Homework-7/Controllers/UserController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Homework_7.Dto;
+using Homework_7.Entity;
+using Homework_7.Mappers;
+using Homework_7.Service;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+
+namespace Homework_7.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserController : ControllerBase
+    {
+        private readonly UserService _userService;
+        public UserController(UserService userService)
+        {
+            _userService = userService;
+        }
+
+        [HttpPost("register")]
+        public IActionResult Register([FromBody] UserRequestDto userDto)
+        {
+            Console.WriteLine($"userDto : {userDto}");
+            Console.WriteLine(JsonConvert.SerializeObject(userDto));
```

Comment:
this line can also be moved to service

Reply:
Used console to debug, forgot to remove it before pushing the code. Will keep in mind now onwards.